src/main/java/io/github/dftrakesh/cloverrest/CloverRestUpdateInventory.java

```diff
@@ -0,0 +1,25 @@
+package io.github.dftrakesh.cloverrest;
+
+import io.github.dftrakesh.cloverrest.handler.JsonBodyHandler;
+import io.github.dftrakesh.cloverrest.model.updateInventory.updateRequest;
+import io.github.dftrakesh.cloverrest.model.updateInventory.updateStockResponse;
+import lombok.SneakyThrows;
+import java.net.URI;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import static io.github.dftrakesh.cloverrest.constantcode.ConstantCodes.V3_ITEM_STOCKS_ENDPOINT;
+import static io.github.dftrakesh.cloverrest.constantcode.ConstantCodes.V3_MERCHANTS;
+
+public class CloverRestUpdateInventory extends CloverRestSDK {
+
+    public CloverRestUpdateInventory(AccessCredentials accessCredentials) { super(accessCredentials);}
+
+    @SneakyThrows
+    public updateStockResponse update(updateRequest inventoryRequest , String itemId ) {
```

Comment:
update method move to CloverRestItem Class and change method name update to "updateInventory"

Reply:
Moved and changed